### DISCUSSION TITLE
Can Java agent report traces to local log files?

### DISCUSSION THREAD

**Discussion by yaojingguo** [apache/skywalking#7976] [Q&A] [answered]
Can Java agent report traces to local log files? So something like Filebeat can be used to monitor these log files and forward the traces to OAP.

Go2sky provides [LogReporter](https://pkg.go.dev/github.com/SkyAPM/go2sky/reporter#NewLogReporter) besides the [GRPCReporter](https://pkg.go.dev/github.com/SkyAPM/go2sky/reporter#NewGRPCReporter).

**Comment by wu-sheng** [comment, marked as answer]
No, there is nothing like this.

**Reply by wu-sheng** [reply]
Go2sky's LogReporter is not something like you asked.

**Reply by yaojingguo** [reply]
I see. Thanks.
